commons/src/test/resources/coverage/cobertura.xml

```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0"?>
+<!--
+
+    SonarQube Apple Plugin - Enables analysis of Swift and Objective-C projects into SonarQube.
+    Copyright © 2022 inside|app (contact@insideapp.fr)
+
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU Lesser General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU Lesser General Public License for more details.
+
+    You should have received a copy of the GNU Lesser General Public License
+    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+-->
+<!DOCTYPE coverage SYSTEM "http://cobertura.sourceforge.net/xml/coverage-04.dtd">
+<coverage line-rate="0.7692307692307693" branch-rate="0.8000000000000000" lines-covered="10" lines-valid="13" branches-covered="4" branches-valid="5" complexity="0.0" timestamp="1650561550" version="Slather 2.7.2">
+  <sources>
+    <source>/Users/ggrousset/Desktop/SQApp</source>
```

Comment:
Ca pointe vers un dossier local de ta machine

Reply:
Right,
However, it does not matter at all (it's a sample cobertura report to unit test parsing).
But I'll fix it anyway !

Reply:
@JessyB done !